aqueduct/metrics/processes.py

```diff
@@ -0,0 +1,25 @@
+from dataclasses import asdict, dataclass
+from typing import Iterable, Tuple
+
+from . import IMetricsItems
+
+
+@dataclass
+class ProcessesStats(IMetricsItems):
+    dead: int = 0
+    running: int = 0
+
+    @property
+    def items(self) -> Iterable[Tuple[str, int]]:
+        return asdict(self).items()
+
+    def extend(self, stats: 'ProcessesStats'):
+        self.dead += stats.dead
+        self.running += stats.running
+
+    def add_dead_process(self):
+        self.dead += 1
+        self.running = 0
```

Comment:
`self.running = 0`
looks like a crutch, can we deal without need `self.running = 0`, maybe we dont need stop metcrics manager when we stoppping flow